README.md

```diff
@@ -33,20 +33,26 @@ soon as we can!
 
 We welcome community contributions!
 
-We use `docker-compose.yml` to quickly and easily provide you with a development environment that mimics real life. Please check it out for helpful hints on how to reach one container from another. To run the end-to-end test, from our home directory:
-- Ensure you have [Docker](https://www.docker.com/) installed with Buildkit support (Docker 18.09 or higher)
-- Run `make test`. That's it!
+To spin up an end-to-end development environment based in Docker:
 
-To drop into a Bash shell into the development environment, run `make dev`. This will automatically run everything you need in order to test and debug your code.
-- Ensure you have the latest authenticator binary: `$ cd authenticator && GOOS=linux GOARCH=amd64 go build && cd ..`.
 - In your local environment, run `$ aws configure` and add an access key and secret.
   - Make sure the access key and secret you configure can assume at least one role.
+- Export an environment variable for the role you're testing with: `$ export TEST_IAM_ROLE=arn:aws:iam::123456789012:role/AssumableRole`.
 - Run `$ make dc-build`.
-- In one window, `$ docker-compose up`.
+- Run `$ docker-compose up`.
 - In another window, `$ make dev`.
-- Export an environment variable for the role you're testing with: `$ export TEST_IAM_ROLE=arn:aws:iam::123456789012:role/AssumableRole`.
 - To use our Python SDK to shoot a request at the authenticator, run
   `$ PGHOST=dbmd5 PGUSER=bob PGDATABASE=db python3 sdk/python/examples/pg2_client.py`.
```

Comment:
@UpGado I toyed with this a little. It no longer works. This doesn't work either:

```
root@374602305b40:/usr/src/approzium# python3 sdk/python/examples/psycopg2.py
Traceback (most recent call last):
  File "sdk/python/examples/psycopg2.py", line 1, in <module>
    import approzium
ModuleNotFoundError: No module named 'approzium'
```

Do you happen to know what the command would currently be?